src/Analyzers/Core/Analyzers/RemoveUnusedParametersAndValues/AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer.SymbolStartAnalyzer.BlockAnalyzer.cs

```diff
@@ -386,7 +386,8 @@ private bool ShouldAnalyze(IOperation operationBlock, ISymbol owningSymbol, ref
                                         // Bail out in presence of OperationKind.None - not implemented IOperation.
                                         if (operation.Kind == OperationKind.None)
                                         {
-                                            hasOperationNoneDescendant = true;
+                                            // `nameof(SomeTypeName)` is a well-known case where operation related to `SomeTypeName` syntax is of kind `None`
+                                            hasUnknownOperationNoneDescendant = operation.Parent is not INameOfOperation;
```

Comment:
@333fred @AlekseyTs do you recall why there is a NoneOperation for this trivial case?

Reply:
> do you recall why there is a NoneOperation for this trivial case?

Do we have a node to represent a type or namespace?